druid-lookups/src/test/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/JdbcExtractionNamespaceCacheFactorySqlLocalQueryTest.java

```diff
@@ -0,0 +1,204 @@
+import io.druid.metadata.MetadataStorageConnectorConfig;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.joda.time.DateTime;
+import org.joda.time.Period;
+import org.junit.Ignore;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.sql.*;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.*;
+import java.util.concurrent.Callable;
+
+/**
+ * Created:         ryanwagner
+ * Date:            2019/06/13
+ *
+ * Class Purpose:   Testing Druid lookup functionality to convert a lookup committed by all nodes
+ *                  into a leader-follower set.
+ *
+ * Leader Role:     Reads data from Oracle & writes to a Kafka topic.
+ * Follower Role:   Reads from Kafka topic & writes back to local Cache.
+ *
+ * Purpose:         These tests, marked as @Ignored, are to be implemented locally to demonstrate full lookup
+ *                  functionality.
+ *
+ * Requirements:    Local instance of mySQL DB, running on the Port noted in the connection String.
+ *                  Local instance of Kafka running on the same port as bootstrap.servers.
+ *                  TODO: Local instance of Druid to load these instances into.
+ *
+ * TODOS:           TODO: Above
+ *                  TODO: Ensure KafkaConsumer runs properly by creating a 1-minute test that periodically
+ *                        writes to an active Kafka topic.
+ */
+public class JdbcExtractionNamespaceCacheFactorySqlLocalQueryTest {
+
+    private HikariConfig config;
+    private String h2dbId = UUID.randomUUID().toString().replace("-", "");
+    private HikariDataSource ds;
+    private Statement jdbcConnection;
+
+    private JDBCExtractionNamespaceCacheFactory jdbcEncFactory = new JDBCExtractionNamespaceCacheFactory();
+    private String jdbcUrl;
+    private String userName;
+    private String passWord;
+    private String jdbcConnectorConfig;
+    private Properties kafkaProperties;
+
+    Date currentMoment = new Date();
+    DateTime currentDateTime = new DateTime(currentMoment);
+    String toDatePlusOneHour = (new SimpleDateFormat("YYYY-MM-dd HH:mm:ss")).format(currentDateTime.plusHours(1).toDate());
+
+    //Mock classes, currently unused in this test suite.
+    @Mock
+    ServiceEmitter serviceEmitter;
+
+    @Mock
+    LookupService lookupService;
+
+    /**
+     * Set up local Kafka Producer and Comsumer, which will share properties.
+     */
+    void setKafkaProperties() {
+        kafkaProperties = new Properties();
+        kafkaProperties.put("bootstrap.servers", "localhost:9092");
```

Comment:
The local query test can use this since all internal tests are all disabled unless the user explicitly sets up Kafka & SQL to use them.  I will set these functions to only be called by their respective tests.